Documentation/doc/scripts/test_doxygen_versions.sh

```diff
@@ -44,7 +47,7 @@ make -j$NB_CORES doc  &> /dev/null
 echo "done."
 cd ../ #scripts
 echo "Creating text files for diff...."
-bash compare_testsuites.sh $PWD/build_doc/doc_output 1> /dev/null
+bash -$- compare_testsuites.sh $PWD/build_doc/doc_output 1> /dev/null
```

Comment:
`$-` is a positional parameter of the shell that contains all the options set by `set`. `-$-` is a way to pass those options to sub-shells.